### DISCUSSION TITLE
Release of Integration test docker networking changes

### DISCUSSION THREAD

**Discussion by lomanm** [quarkusio/quarkus#23561] [Q&A]
I am wondering when PR [#23543](https://github.com/quarkusio/quarkus/pull/23543) will be released.  It contains a fix that will allow my team to run containerized IT tests.  Changes in version 2.6.x (described in this [issue](https://github.com/quarkusio/quarkus/issues/23531)) broke our ability to run the IT tests.  So I can't take advantage of new features in Quarkus 2.6+ until this PR gets released.

It would be great if this fix could be added to the next 2.7 release.  Thanks!

**Comment by famod** [comment]
@geoand has added the `triage/backport?` label, meaning that it will most likely be backported to 2.7.2 (whenever that will be released).

**Comment by gsmet** [comment]
I plan to release 2.7.2.Final on February 21st.
